### ISSUE TITLE
Add extension to open-vsx

### ISSUE THREAD

**Issue by pm4rcin** [Kirides/vscode-daedalus#4]
Hi, I think you should consider adding this extension to [open-vsx.org](https://open-vsx.org) so people who use e.g. VSCodium can install it easily because visual studio marketplace is disabled there.

**Comment by Kirides** [comment]
This extension is not available on VS marketplace.The only place to get it is the releases tab currently.

**Comment by pm4rcin** [comment]
You're right. I confused it with another extension. But I hope it'll be released on open-vsx one day :)

**Comment by Patrix9999** [comment]
> This extension is not available on VS marketplace.The only place to get it is the releases tab currently.

Do you plan to add this extension to vs code marketplace?

**Comment by kirides** [comment]
I plan to, but had no internet for quite a while now.

Will get to it in a while.
